.gitignore

```diff
@@ -1,2 +1,19 @@
+logs
+*.log
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+lerna-debug.log*
```

Comment:
@PCalape No need to ignore these we are not using Yarn and Lerna so we will never have these files.

Reply:
Copy on this @arnellebalane. Will apply next commit